apps/camera/js/views/viewfinder.js

```diff
@@ -161,20 +161,18 @@ module.exports = View.extend({
     }, this.fadeTime);
   },
 
-  fadeIn: function(ms) {
+  fadeIn: function(first) {
     debug('fade-in');
     var self = this;
-    if (typeof ms !== 'number') { ms = this.fadeTime; }
-    if (ms) {
-      this.el.style.transitionDuration = ms + 'ms';
-      this.show();
-      this.fadeTimeout = setTimeout(function() {
-        self.el.style.transitionDuration = '';
-        self.emit('fadedin');
-      }, ms);
+    this.show();
+    if (first) {
+      self.emit('fadedin');
+      // Delay to prevent the duration from applying too early
+      setTimeout(function() {
```

Comment:
try wrapping in `requestAnimationFrame(() => { ... })` instead (you also won't need `var self = this;` if you do that)